plugin/index.php

```diff
@@ -19,13 +23,26 @@ function gutenberg_menu() {
 	);
 }
 
+// Hook.
+add_action( 'admin_enqueue_scripts', 'gutenberg_scripts_and_styles' );
```

Comment:
I am not sure if there is a pattern there. I myself follow both, but I remain consistent at least in one single project.

It makes lesser sense as two block level comments stacked together do NOT look good. That's the only reason why I do this.

Normally, I creat  a function like this. 
![](https://i.imgur.com/vsFGcUt.png)